.github/workflows/nightly_release.yml

```diff
@@ -0,0 +1,69 @@
+# Helpful YAML parser to clarify YAML syntax:
+# https://yaml-online-parser.appspot.com/
+
+name: Build and Release (nightly)
+
+on:
+  workflow_run:
+    workflows: [Unit Tests (nightly)]
+    types:
+      - completed
+  workflow_dispatch: {}
+
+jobs:
+  check-tests-pass:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'workflow_dispatch' || github.event.workflow_run.conclusion == 'success'
+    steps:
+      - run: echo 'The triggering workflow passed'
+
+  build-and-release-nightly:
+    needs: check-tests-pass
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Get and set nightly date
+        id: date
+        run: |
+          DATE=$(date +'%Y%m%d')
+          echo "NIGHTLY_RELEASE_DATE=${DATE}" >> $GITHUB_ENV
+          echo "date=${DATE}" >> $GITHUB_OUTPUT
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      - name: Install python-build and twine
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install build twine wheel
+          python -m pip list
+
+      - name: Build the wheel
+        run: |
+          python setup.py bdist_wheel
+
+      - name: Verify the distribution
+        run: twine check --strict dist/*
+
+      - name: List the contents of ai_edge_torch wheel
+        run: python -m zipfile --list dist/ai_edge_torch*.whl
+
+      - name: Get ai_edge_torch wheel version
+        id: wheel_version
+        run: |
+          WHEEL=$(ls -t dist/ai_edge_torch*.whl | head -n 1)
+          VERSION=$(python -c "from pkginfo import Wheel; print(Wheel(r'${WHEEL}').version)")
+          echo "version=${VERSION}" >> $GITHUB_OUTPUT
+
+      - name: Upload release to github
+        uses: softprops/action-gh-release@v2
+        with:
+          files: |
+            dist/*.whl
+            requirements.txt
+          prerelease: true
+          tag_name: v${{ steps.wheel_version.outputs.version }}
```

Comment:
I was wondering if we should skip uploading the nightly artifacts on github, and instead only have the nightly pip package. And keep https://github.com/google-ai-edge/ai-edge-torch/releases for manual releases only.

Likely worth a chat before we decide one way or the other.

Reply:
Releasing to pypi only should be fine. I set this up while considering to keep requirements.txt in github releases, but that should be minor and we won't expect people to install old nightly versions.

Reply:
Updated to upload the nightly to pypi. Please take a look again.